### DISCUSSION TITLE
Does roaming over access points cause errors?

### DISCUSSION THREAD

**Discussion by NikolaasD** [SafeExamBrowser/seb-win-refactoring#1058] [Q&A]
Today I noticed a rise in student who unexpectedly got the red screen while taking an exam with Moodle.  For some students that page would not respond. Some students got to a page that wouldn't load after the correct password was entered.

In our buildings, almost 90% of the rooms have their own Access Point (Ruckus). They are controlled in the cloud. When there are too many clients in one room, they'll try to distribute traffic to nearby AP's. Today noticed one AP went offline and rebooted a while later. This means everyone on that access point was redistributed to its neighbours?

Would the Safe Exam Browser detect this and see it as a problem?

**Comment by dbuechel** [comment]
No, that is a network / infrastructure issue and as long as the wireless network including SSID and credentials remain the same, SEB does neither detect that nor should this cause any problems.
